### DISCUSSION TITLE
Étude de cas : "Ethnographie" P24

### DISCUSSION THREAD

**Discussion by KoffiBalayn** [Hypertopic/HyperGlosae#214] [Show and tell]
Here is the video of our work on the Ethnografiy's subject on Hyperglosae.

https://github.com/Hypertopic/HyperGlosae/assets/160734127/09432543-5c75-4dd5-8578-e886a5a995d7

**Comment by PElbaz** [comment]
Les consultations avec la partie prenante sont intéressantes.
Je n’ai pas pu entendre la fin lors du premier visionnage sur IPad , le son s’est retiré, j’ai dû redémarré la vidéo.

**Comment by Mcoeuret** [comment]
Bonjour à vous tous,

Je vous félicite pour l’avancée d’Hyperglosae et l’implémentation de ces nouvelles fonctionnalités importantes pour l’amélioration du parcours utilisateur.

**Remarques générales sur le projet**

La présentation du projet est claire et les définitions sont en accord avec les usages actuels. Les fonctionnalités demandées ont été mises en place avec succès. Concernant les modifications proposées, vous avez suggéré des pistes intéressantes qui favorisent la lisibilité de l'interface.

**Vous trouverez ci-dessous quelques commentaires relatifs à de futures améliorations possibles :**

1:15 : En ce qui concerne l’utilisation de la terminologie « chemin naturel/processus naturel de la pensée », il est important de noter qu'il n'existe pas de chemin naturel. Un des enjeux majeurs de la conception d’un logiciel est de proposer un parcours utilisateur intuitif.

1:25: Je vous félicite pour avoir intégré mes remarques sur la forme et la superposition des boutons cliquables. Cependant, faites attention à ne pas couper le texte, car cela nuit à la lisibilité. Par exemple, à 9:13, les boutons ne sont plus homogènes (forme et superposition), il faut éviter d’implémenter un bouton dans un autre.

3:45: Très bonne gestion du feedback concernant l’échec de modification du document sans connexion utilisateur.

4:23: Attention, cliquer à l’extérieur de la glose pour valider un commentaire n’est pas intuitif. Il pourrait être envisagé de mettre un icône de validation.

6:50: Je vous félicite pour la fonctionnalité d’étiquetage (formulation de l’étiquette, modification, repère dans le texte et affichage).
Cette fonctionnalité permettra à l’utilisateur de mieux se repérer dans son travail d’analyse.

Margaux

**Comment by benel** [comment]
@antooinex @GwenRiou @KoffiBalayn @mbel-jordan @Fluorite-Eyes
Vous avez des messages :)
